tensorflow-core/tensorflow-core-api/src/gen/annotations/org/tensorflow/op/Ops.java

```diff
@@ -8143,6 +8076,26 @@ public Ops withSubScope(String childScopeName) {
     return new Ops(scope.withSubScope(childScopeName));
   }
 
+  /**
+   * Returns an API that builds init operations.
+   * <p>Init operations will be initialized at session creation, will have their inputs (and control inputs) made init ops as well, and are never used as control dependencies.
+   * Additionally, this scope drops all of its control dependencies.  If an input can not be made an init op (i.e. a Placeholder), will error on op creation.
+   */
+  public Ops initScope() {
```

Comment:
It's `initScope` in Python and I didn't want to name it the same as the op it's replacing (could cause silent failures), but I could.

Reply:
yeah, in fact ideally it would match the name pattern used for other endpoints of `Ops` that starts a new scope, e.g. `withSubScope`, `withName`, `withDevice`, etc. I'm not sure I like it be following this logic, it would be `withInit` or `withInitScope`? Not sure it makes sense...

Reply:
`withInitScope` seems fine to me.